src/kw_env_show.sh

```diff
@@ -0,0 +1,94 @@
+include "${KW_LIB_DIR}/kwlib.sh"
+include "${KW_LIB_DIR}/kwio.sh"
+include "${KW_LIB_DIR}/kw_string.sh"
+
+declare -gA options_values
+
+function env_show_main()
+{
+  parse_env_show_options "$@"
+
+  if [[ -n "${options_values['INSTALL']}" ]]; then
+    install_home
+    return "$?"
+  fi
+}
+
+function install_home()
+{
+  say 'Installing ...'
+
+  if [[ -f "$HOME/.bashrc" ]]; then
+    update_path ".bashrc"
+  else
+    warning ".bashrc file not found."
+  fi
+}
+
+function update_path()
+{
+  local shellrc="$1"
+
+  local content
+
+  content=$(
+    cat << 'EOF'
+parse_git_file() { git_file=$(git rev-parse --show-toplevel 2>/dev/null)/.kw/env.current; [ -f "$git_file" ] && printf '[%s]' "$(cat "$git_file")"; }
+
+PS1='\[\e[0;34m\]\u@\h \[\e[0;32m\]\w\[\e[0;36m\] $(parse_git_file)\[\e[0m\] $ '
```

Comment:
Maybe we can create a file with this code in the etc/ folder and load it when we need it.